src/main/java/com/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,38 @@
+package com.duke.command;
+
+import com.duke.task.Task;
+import com.duke.task.TaskList;
+import com.duke.util.Storage;
+import com.duke.util.Ui;
+
+public class AddDeadlineCommand extends Command {
```

Comment:
Since `AddDeadlineCommand`, `AddEventCommand` and `AddToDoCommand` have the same functionalities, would it be better to have a common class `AddTaskCommand` instead?